client/a8c-for-agencies/components/offering/style.scss

```diff
@@ -0,0 +1,92 @@
+@import "@wordpress/base-styles/breakpoints";
+@import "@wordpress/base-styles/mixins";
+
+
+.a4a-offering-card__wrapper {
+	padding-top: 16px;
+
+	.a4a-offering-card__title {
+		font-weight: 500;
+		font-size: 1rem;
+		line-height: 24px;
+		color: var(--studio-gray-100);
```

Comment:
We've learned recently that it is recommended to avoid using `--studio` variables directly in favor of the Theme properties. (e.g. --color-neutral-100). This allows us to easily override color schemes without modifying the CSS code.


https://wpcalypso.wordpress.com/devdocs/docs/color.md

Reply:
Thank you for pointing it out. Haven't heard about it before.